java/arcs/core/data/proto/manifest.proto

```diff
@@ -317,9 +317,19 @@ message PolicyConstraintsProto {
 
   InformationFlowLabelProto.Predicate egress_check = 2;
 
-  message StoreConstraints {
-    string store_id = 1;
-    repeated ClaimProto claims = 2;
+  message PartialClaim {
+    repeated AccessPathProto.Selector selectors = 1;
+    InformationFlowLabelProto.Predicate predicate = 2;
   }
-  repeated StoreConstraints store_constraints = 3;
+
+  message TypeClaims {
+    // The name of the schema the claims apply to.
+    string schema_name = 1;
+
+    // Partial claims (Assume claims without an AccessPath root).
+    repeated PartialClaim partial_claims = 2;
```

Comment:
We could do that. However, the DFA code wouldn't understand that type of AccessPath root, and would need to convert it into a handle-based root. You also wouldn't be able to express such a claim in the manifest, it would only be constructable by the policy translator. 

It also seems a bit misleading to have something like `Claim(AccessPath.Root.SchemaName("Foo"), ...)`, but not have it apply to all handles with a `Foo` schema, only the specific ingress ones. I dunno, this felt more like a policy thing than a general DFA thing to me. 

I thought it was better to leave the Claim (and AccessPath) class for real claims, and just use something else here. If you think Claim is better here I'm happy to delete PartialClaim and use Claim instead. It'll be less code, but I worry that it's harder to understand.

Reply:
Sounds good. Can we name it something else though? `PartialClaims` makes me wonder what is partial. How about `FieldClaims` or `FieldPathClaims` or `SelectorClaims`?

Also, `TypeClaims` to `SchemaClaims`?

Reply:
Done